packages/calcite-components/src/components/block/block.tsx

```diff
@@ -350,6 +399,8 @@ export class Block
             class={CSS.content}
             hidden={!open}
             id={regionId}
+            // eslint-disable-next-line react/jsx-sort-props
```

Comment:
Can you add the explainer from https://github.com/Esri/calcite-design-system/pull/7584? 

> // eslint-disable-next-line react/jsx-sort-props -- ref should be last so node attrs/props are in sync (see https://github.com/Esri/calcite-design-system/pull/6530)

Applies to other `ref`s.